Mathlib/Probability/Kernel/Composition/MeasureComp.lean

```diff
@@ -0,0 +1,137 @@
+/-
+Copyright (c) 2025 Rémy Degenne. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Rémy Degenne, Lorenzo Luccioli
+-/
+import Mathlib.Probability.Kernel.Composition.MeasureCompProd
+
+/-!
+# Lemmas about the composition of a measure and a kernel
+
+This operation, for which we introduce the notation `∘ₘ`, takes `μ : Measure α` and
+`κ : Kernel α β` and creates `κ ∘ₘ μ : Measure β`. The integral of a function against `κ ∘ₘ μ` is
+`∫⁻ x, f x ∂(κ ∘ₘ μ) = ∫⁻ a, ∫⁻ b, f b ∂(κ a) ∂μ`.
+
+This file does not define composition but only introduces notation for
+`MeasureTheory.Measure.bind μ κ`.
+
+## Notations
+
+* `κ ∘ₘ μ = MeasureTheory.Measure.bind μ κ`, for `κ` a kernel.
+-/
+
+open scoped ENNReal
+
+open ProbabilityTheory MeasureTheory
+
+namespace MeasureTheory.Measure
+
+variable {α β γ : Type*} {mα : MeasurableSpace α} {mβ : MeasurableSpace β} {mγ : MeasurableSpace γ}
+  {μ ν : Measure α} {κ η : Kernel α β}
+
+/-- Composition of a measure and a kernel.
+
+Notation for `MeasureTheory.Measure.bind` -/
+scoped[ProbabilityTheory] notation3 κ " ∘ₘ " μ:100 => MeasureTheory.Measure.bind μ κ
```

Comment:
Would it make sense to add the notation earlier, and use it to state some preexisting lemmas like `IsProper.setLIntegral_eq_bind` (or move this lemma to the current file?)